manifests/repositories.pp

```diff
@@ -20,8 +20,8 @@
         }
       } else {
         yumrepo { 'htcondor-stable':
-          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${major_release}",
-          baseurl  => "http://research.cs.wisc.edu/htcondor/yum/stable/rhel${major_release}",
+          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${facts['os']['release']['major']}",
+          baseurl  => 'http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever',
```

Comment:
That is interesting, thank you for the explanation.
I am using 

```
cat /etc/yum.conf | grep distrover
distroverpkg=redhat-release

yum info centos-release | grep Version
Version     : 7
[root@hd58 ~]# cat /etc/centos-release
CentOS Linux release 7.3.1611 (Core) 

[root@hd58 ~]# cat /etc/redhat-release 
CentOS Linux release 7.3.1611 (Core) 


python -c 'import yum, pprint; yb = yum.YumBase(); pprint.pprint(yb.conf.yumvar, width=1)'
{'arch': 'ia32e',
 'basearch': 'x86_64',
 'infra': 'stock',
 'releasever': '7.3',
 'uuid': '55cd454b-8c2c-446f-b28d-bbbea5aa61bb'}
```

After changing `distroverpkg=redhat-release` to `distroverpkg=centos-release` things work as expected. I wonder how my system came into this configuration - likely I copied yum.conf.

On well-configured systems this should not be a problem.

Long-term I will implement more tests for this puppet module (issue #68) to automatically verify the validity.

I will merge it now.